railties/lib/rails/generators/rails/scaffold_controller/templates/controller.rb

```diff
@@ -49,7 +77,27 @@ def destroy
     @<%= orm_instance.destroy %>
     redirect_to <%= index_helper %>_url, notice: <%= "'#{human_name} was successfully destroyed.'" %>
   end
+  <% if collection_routing? %>
+  # Collection routes
+  # PATCH <%= route_url %>/1..10
+  def update_many
+    <%= orm_class.find(class_name, "#{singular_table_name}_ids") %>.each do |<%= singular_table_name %>|
+      <%= singular_table_name %>.update(params[:<%= plural_table_name %>][:"#{<%= singular_table_name %>.id}"])
+    end
+    redirect_to @<%= singular_table_name %>, notice: <%= "'#{human_name}s were successfully updated.'" %>
+  end
 
+  # PUT <%= route_url %>/1..10
+  def replace
+    <%= orm_class.destroy_all(class_name) %>
+    create
+  end
```

Comment:
PUT on collection could be used to reset state such as resetting the set of questions in a quiz or reloading a fresh set of test data in an application that acts as a test data server. More importantly this completes the HTTP verbs for our case. It might be rarely used but it's there in case you need to. Can't think of anything else straight out of my mind. Maybe @pixeltrix can help here.